packages/cypress-angular-dev-server/src/lib/create-angular-webpack-config.ts

```diff
@@ -30,13 +30,12 @@ export async function createAngularWebpackConfig(config: {
       /* @hack outputPath is required, otherwise `getCommonConfig` crashes. */
       outputPath: '',
       index: null,
-      main: null,
+      /* main is required, otherwise AngularWebpackCompiler is not executed. */
+      main: getSystemPath(resolve(normalize(config.sourceRoot), normalize('main.ts'))),
+      polyfills: getSystemPath(resolve(normalize(config.sourceRoot), normalize('polyfills.ts'))),
```

Comment:
I guess that this would only work if it's an app and would crash if it's a library as a library wouldn't have a `main.ts` or `polyfill.ts`.
The only solution I can see right now is  implementing the @todo below 😊 . Cf. https://github.com/jscutlery/test-utils/issues/51

Reply:
Good point.